src/Modal.tsx

```diff
@@ -110,7 +125,7 @@ export class Modal extends Component {
   };
 
   hide = () => {
-    this.setState({ animationDirection: "out" }, () => {
+    this.setState({ currentAnimation: "out" }, () => {
```

Comment:
I assumed this was a typo.

Reply:
😅 ouch? I thought the animation out was working though -- weird